host_guest/GDCC_and_guests/GDCC_submission.txt

```diff
@@ -0,0 +1,90 @@
+# Results for GDCC
+#
+# This file will be automatically parsed.  It must contain the following five elements: predictions, name of method, software listing, method, and ranked.
+# These elements must be provided in the order shown.
+# Please explicitly describe how you handle ions and pKa effects.
+#
+# The data in each prediction line should be structured as follows, with all (up to six) numbers in kcal/mol.
+# host-guest ID (note that the host varies!), Free energy, free energy SEM, free energy method error, enthalpy, enthalpy SEM, enthalpy method uncertainty
+# Note that the "method error" should be your estimate of ACCURACY of this particular approach
+# for the compound considered.
+# Order does not matter.
+# The list of predictions must begin with the "Prediction:" keyword, as illustrated here.
+Predictions:
+OA-g1, -10.1,,,  -0.3,,
+OA-g2, -10.5,,,  -20.5,,
+OA-g3, -5.4,,,   -2.9,,
+OA-g3, -2.3,,,   -5.0,,
+OA-g4, -8.9,,,   -12.1,,
+OA-g5, -1.4,,,   -1.8,,
+OA-g6, -9.2,,,   -0.8,,
+OA-g7, -2.3,,,   -5.0,,
+OA-g8, -8.9,,,   -12.1,,
+exoOA-g1, -1.4,,,   -1.8,,
+exoOA-g2, -6.2,,,   -0.4,,
+exoOA-g3, -8.2,,,   -0.3,,
+exoOA-g4, -7.2,,,   -0.6,,
+exoOA-g5, -8.9,,,   -12.1,,
+exoOA-g6, -1.4,,,   -1.8,,
+exoOA-g7, -9.2,,,   -0.8,,
+exoOA-g8, -2.3,,,   -5.0,,
+#
+# Please provide an informal yet informative name of the method used.
+# Following is sample text; please edit to your taste.
+# The "Name:" keyword is required, as shown here.
+Name:
+Long/DDM/GAFF/TIP3P
+#
+# All major software packages used and their versions
+# Following is sample text; please edit to your taste.
+# The "Software:" keyword is required.
+Software:
+Gromacs 5.0.2
+Gaussian 09
+#
+# Methodology and computational details.
```

Comment:
Just a thought since this came up with the last challenge-- if there is anything in particular that might come in handy for comparing to reference calculations down the road, it might be good to explicitly mention it here (ex. water model, equilibration amount, production amount, etc.). Also, not sure how to handle this exactly, but there have been cases where all the sentences in the methods section aren't cropped at even points and it makes it hard to read (ie you have to continuously scroll right for some of the sentences). Maybe mentioning it here would help a little.

Reply:
If you have any ideas on what to have them specify that works across all categories of methods let me know. I added pKa, protonation state stuff now, but other than that I don't know what to tell everyone since what's important depends on the method. :(